fw/ctrl/serial.c

```diff
@@ -123,7 +123,7 @@ char *next_word(const char *c)
   while ((*c!=0) && (*c!=' ')) c++;
   while (*c==' ') c++;
   if (*c==0) return NULL;
-  else return c;
+  else return (char *)c;
```

Comment:
Returning a const seems odd, but I didn't bother to see what was going on here.  Suppress GCC complaining.